content/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/reverting-a-pull-request.md

```diff
@@ -33,6 +33,6 @@ For more information about using Git to manually revert individual commits, see
 
 {% data reusables.repositories.sidebar-pr %}
 2. In the "Pull Requests" list, click the pull request you'd like to revert.
-3. Near the bottom of the pull request, click **Revert**.
+3. Near the bottom of the pull request, click **Revert**. If **Revert** does not show up as an option, you will need to ask for write permissions.
```

Comment:
Maybe say "displayed" rather than "show up":
```suggestion
3. Near the bottom of the pull request, click **Revert**. If the **Revert** option isn't displayed, you'll need to ask the repository administrator for write permissions.
```

Reply:
This suggestion makes it much more clear and sounds better. Thank you again!